tests/unit/specs/components/transactions/TransactionList.spec.js

```diff
@@ -12,7 +12,7 @@ describe(`TransactionList`, () => {
   let height = 1086769
   let oneDay = 86400 // One day
   let offset = 86400 / 7 // One day / 7s block time
-  let timestamp = new Date().getTime() // Today
+  let timestamp = new Date('2019-11-27T04:41:20') // cannot do new Date().getTime() because of snapshots
```

Comment:
@Bitcoinera `new Date()` is fixed to a certain time in `fixed_time.js`

Reply:
BTW Would be nice to set a different date in fixed_time.js to allow susbtract at least 1 day, 1 month and 1 year to allow categorization by date as in transactions page.

Reply:
Oh, I see. Good to know :ok_hand: